src/axis-label.js

```diff
@@ -26,8 +26,14 @@ const AxisLabel = ({
   arrow = true,
   align = 'right',
 }) => {
-  const { x, y, pl, pr, pt, pb, apl, apr, apt, apb } = useChart()
```

Comment:
assume x and y weren't being used here, but just double checking! from a quick look i don't see them used

Reply:
Yup they were unused!